packages/developer-portal/src/types/global.d.ts

```diff
@@ -17,6 +17,7 @@ export type Config = {
   swaggerUri: string
   PUSHER_KEY: string
   DEPLOYMENT_SERVICE_HOST: string
+  pipelineWhitelist?: string[]
```

Comment:
Should be required field, you need to add it to the /core/index.tsx too where the config gets instantiated

Reply:
oh ok, and force a default value of [] 👍🏼

Reply:
Yep, nice one